File: codewars/DeodorantEvaporator/Evaporator.java
public class Evaporator { 
	
	public static int evaporator(double content, double evap_per_day, double threshold) {
    double thresholdContent = content * threshold / 100;
    int count = 0;
    
    while (content >= thresholdContent) {
      content = content - (content * evap_per_day / 100);
      count += 1;
    }
    
		return count;
	}
}

File: codewars/DeodorantEvaporator/Evaporator.py
def evaporator(content, evap_per_day, threshold):
    count = 0
    thresholdContent = (threshold/100) * content
    
    while content >= thresholdContent:
        content = content - ((evap_per_day / 100) * content)
        count += 1
    
    return count